README.md

```diff
@@ -1,6 +1,11 @@
 [![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)
 
 # Barefoot Nomad - Making company travel and accomodation easy and convenient.
+Barefoot Nomad - Making company travel and accomodation easy and convinient.
+=======
+[![Coverage Status](https://coveralls.io/repos/github/andela/team-odd-bn-backend/badge.svg?branch=develop)](https://coveralls.io/github/andela/team-odd-bn-backend?branch=develop)
+[![Maintainability](https://api.codeclimate.com/v1/badges/97cfd183a562e8b5d53f/maintainability)](https://codeclimate.com/github/andela/team-odd-bn-backend/maintainability)
```

Comment:
Good work @nziokaivy
Nevertheless, would you mind taking a look at:
1. aligning both **coverage & code climate** badges together with **houndCI badge**.
2. Getting rid of duplicated project description in README file
<img width="1025" alt="Screen Shot 2019-11-06 at 22 18 35" src="https://user-images.githubusercontent.com/35301129/68334483-67f66a80-00e3-11ea-9418-27370ca2b490.png">

Reply:
@Victor Noted. Working on it